mautic/mautic
# Update StageBundle for Symfony 3

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | N
| Automated tests included? | N
| Related user documentation PR URL | N/A
| Related developer documentation PR URL | N/A
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/7999
| BC breaks? | N/A
| Deprecations? | N/A

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

Upgrades stage bundle for Symfony 3

[//]: # ( As applicable: )

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Ensure Stage creation works
3. Ensure when adding stages to a campaign action, they are displayed correctly in the list
4. Ensure contact edit form works and you can select stages (both full and quick add)
5. Ensure contact preferences page works with stages
